src/controller/python/chip/clusters/Attribute.py

```diff
@@ -719,22 +720,33 @@ def _handleSubscriptionEstablished(self, subscriptionId):
             logging.info("Re-subscription succeeded!")
             if self._subscription_handler._onResubscriptionSucceededCb is not None:
                 if (self._subscription_handler._onResubscriptionSucceededCb_isAsync):
-                    self._event_loop.create_task(
-                        self._subscription_handler._onResubscriptionSucceededCb(self._subscription_handler))
+                    if self._event_loop.is_closed():
+                        asyncio.run(
```

Comment:
@caubut-charter In this PR, we have added resubscription test in python, where it overrides the default liveness timeout, I am not sure why this test does not capture this issue, maybe we can update this test to automate this part.

https://github.com/project-chip/connectedhomeip/pull/20080/files

Reply:
Ah, let me take a look tomorrow.  I think I introduced a bug due to reason below.

https://docs.python.org/3/library/asyncio-task.html#asyncio.run
> This function cannot be called when another asyncio event loop is running in the same thread.

The test you linked runs everything inside a loop and won't hit the issue I am seeing.

I am hitting the issue calling [ZCLSubscribeAttribute](https://github.com/project-chip/connectedhomeip/blob/93d72d265b449488ed260264620a6344a40550c6/src/controller/python/chip/ChipDeviceCtrl.py#L1083) which uses a short lived loop which triggers the error I am seeing.